spec/jobs/delete_missed_reservations_job_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'spec_helper'
+
+describe DeleteMissedReservationsJob, type: :job do
+  it 'deletes appropriate reservations' do
+    mock_app_config(res_exp_time: 5)
+    FactoryGirl.create(:missed_reservation,
+                       start_date: Time.zone.today - 11.days,
+                       due_date: Time.zone.today - 10.days)
+    missed = FactoryGirl.create(:missed_reservation,
+                                start_date: Time.zone.today - 4.days,
+                                due_date: Time.zone.today - 3.days)
+    valid = FactoryGirl.create(:valid_reservation)
+    described_class.perform_now
+    expect(Reservation.all).to match_array([missed, valid])
```

Comment:
While I'm not sure this would be better, as an alternative to relying on a query to test this couldn't we use stubs and check that the correct objects receive `delete` or `destroy` and that the others don't? I just feel like using a query to test for an outcome can get you into trouble since it's very dependent on having a clean database to start with.